projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.module.spec.ts

```diff
@@ -55,3 +55,21 @@ describe('NgxSkeletonLoaderModule.forRoot() method', () => {
     expect(console.info).toHaveBeenCalledTimes(0);
   });
 });
+
+describe('NgxSkeletonLoaderModule.forRoot() method', () => {
```

Comment:
Same for this one... If anyone has a better idea and has experience with Jasmine, I'd be happy to commit new changes!

Reply:
All the other scenarios are covered on the component tests, so this one is just to make sure we are following the the rules in a module level. In that case, you don't need to create another describe. Instead, you can have the beforeEach configuring the module via `forRoot()` combined and updating the assertion to check for the config itself (in this case, `appearance` and `count`).

Reply:
@willmendesneto There's something I feel like I don't understand here. Why is it possible to import the modules without calling the forRoot method? I mean like... what is the difference between importing `NgxSkeletonLoaderModule` and `NgxSkeletonLoaderModule.forRoot()` ?

Also, as @dev04 mentioned, what happens when calling the forRoot method in several places in the same application? I don't think the module's constructor actually prevents that.
> What happens if someone tries to use `.forRoot(...)` in multiple places? Is it a thing that we should care or it doesn't even matter?

I've seen in some other libraries as well as in articles around the web and in the official documentation that the forRoot pattern is finding its root in the singleton pattern and that you actually should prevent to generate multiple instances of the same module. 

I think we should add constraints to how the module can be added to the application, as well as enforcing the forRoot pattern. I can do this in another PR but this would undoubtely be a breaking change and should move the package from v2.9.2 to v3.0.0 whereas this PR is aiming to be v2.10.0.

Source: 
- https://angular.io/guide/singleton-services#the-forroot-pattern 
- https://www.pluralsight.com/guides/forroot-and-forchild-static-methods-on-the-angular-router
- https://generic-ui.com/blog/angular-module-for-root-pattern

Reply:
At this module, `forRoot()` is used as an optional configuration to help consumers, so if the consumers need to change something on the module configuration, they can do it by forRoot as well. If `forRoot()` has been called multiple times by misconfiguration on the consumer's side the solution at the moment is to have them revisiting their configuration, but I can share in advance that was never raised as an issue - so this is an issue that's not an issue at the moment. 

Bundle size is a relevant concern on this package, so the added features are available based on the issues consumers are raising and other requirements. If it comes as an issue that can be discussed and addressed accordingly, but at the moment is not the case.

In case we discuss and move forward of having some error message for it, this should be done only in dev environment, like the errors applied for attributes and misconfiguration. That could be done in that case is to have a prevent multiple `forRoot()` calls (https://angular.io/guide/singleton-services#prevent-reimport-of-the-greetingmodule).

I see your point, but it's good to mention that's not related to this pull request, so it's good to have that covered in another pull request and discussed in an issue, so we can have some shared knowledge with the module consumers.

Reply:
Described in #81, we can continue discussing this overthere.